test/utils.sh

```diff
@@ -176,3 +176,51 @@ handle_error()
     exit 0
   fi
 }
+
+
+# The function will be used by the tekton tasks of build-definitions
+# It returns a map of manifests in an image index received by running `skopeo inspect --no-tags --raw` on an image
+get_image_index_manifests() {
+  local image_url=""
+
+  #Usage information
+  get_image_index_manifests_usage()
+  {
+    echo "
+  get_image_index_manifests  -i IMAGE_URL
+  " >&2
+    exit 2
+  }
+
+  local OPTIND opt
+  while getopts "i:" opt; do
+      case "${opt}" in
+          i)
+              image_url="${OPTARG}" ;;
+          *)
+              get_image_index_manifests_usage ;;
+      esac
+  done
+  shift $((OPTIND-1))
+
+  if [ -z "${image_url}" ]; then
+    echo "Missing parameter: -i IMAGE_URL" >&2
+    exit 2
+  fi
+
+  local skopeo_command="skopeo inspect --no-tags --raw"
+  skopeo_command+=" docker://${image_url}"
+
+  if ! command_output=$(eval "$skopeo_command" 2>&1); then
```

Comment:
sorry I haven't noticed before but for this `2>&1` I don't think that this would work, it will merge random errors/warnigns/whatever into resulting JSON and fail horribly. We have to keep STDOUT and STDERR separate.

skopeo can print lot of warnings inot STDERR sometimes